crates/notion-core/src/tool/mod.rs

```diff
@@ -174,3 +199,214 @@ fn intercept_global_installs() -> bool {
     // We should only intercept global installs if the NOTION_UNSAFE_GLOBAL variable is not set
     env::var_os(UNSAFE_GLOBAL).is_none()
 }
+
+#[cfg(test)]
+mod tests {
+    mod try_from_str {
```

Comment:
The extra module here makes the test output look nice: `tools::tests::try_from_str::parses_bare_node`, etc.